src/server/main_service.cc

```diff
@@ -1234,7 +1234,7 @@ void Service::DispatchCommand(ArgSlice args, SinkReplyBuilder* builder,
 
   dfly_cntx->cid = cid;
 
-  if (!InvokeCmd(cid, args_no_cmd, builder, dfly_cntx)) {
+  if (!InvokeCmd(cid, args_no_cmd, builder, dfly_cntx, cmd)) {
```

Comment:
this might break dual word registered commands like `ACL ...`

Reply:
changed so that only aliases are handled specially when counting stats, all other commands use `cmd.name`, by checking if the passed in command was part of alias map